modules/account-lib/src/coin/hbar/transactionBuilderFactory.ts

```diff
@@ -27,6 +33,9 @@ export class TransactionBuilderFactory extends BaseTransactionBuilderFactory {
     const tx = this.parseTransaction(raw);
     switch (tx.txBody.data) {
       case 'cryptoTransfer':
+        if (tx.txBody.cryptoTransfer && isTokenTransfer(tx.txBody.cryptoTransfer)) {
```

Comment:
You could make this cleaner and just write `return (isTokenTransfer(tx.txBody.cryptoTransfer) ? this.getTokenTransferBuilder(tx) : this.getTransferBuilder(tx)` if you allow `isTokenTransfer` to accept `null` values as well.